test/resources/microshift-host.resource

```diff
@@ -55,3 +55,17 @@ Reboot MicroShift Host
     Sleep    30s
     Wait Until Keyword Succeeds    5m    15s
     ...    SSH Connection To MicroShift Host Should Be Functional
+
+Create Thin Storage Pool
+    [Documentation]    Create a new thin storage pool
+    ${stderr}    ${rc}=    SSHLibrary.Execute Command    lvcreate --size 4G --thin /dev/rhel --name thin
```

Comment:
why do we need `sshLibrary` here, why not `OperatingSystem.Execute Command` like above?